types/alks-node/index.d.ts

```diff
@@ -0,0 +1,125 @@
+// Type definitions for alks-node 0.9
+// Project: https://github.com/Cox-Automotive/alks-node#readme
+// Definitions by: Matt Hoang <https://github.com/matthoang08>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Moment } from 'moment';
+
+export interface Auth {
+    token?: string;
+    password?: string;
+}
+export interface Account {
```

Comment:
`sessionTime` and `password` are missing. Also based on `refreshTokenToAccessToken` looks like some of these options are optional.

Reply:
Documentation is outdated: function definition here:
https://github.com/Cox-Automotive/alks-node/blob/b2a951245421ada5ba0c856e2c167c5808c16025/lib/alks-api.js#L601